test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -400,18 +352,21 @@ int UcxContext::is_timeout_elapsed(struct timeval const *tv_prior, double timeou
 ucs_status_t UcxContext::epoll_init()
 {
     ucs_status_t status;
+    epoll_event  ev;
 
-    if (_use_epoll != true) {
+    if (!_use_epoll) {
         return UCS_OK;
     }
 
+    _epoll_fd = epoll_create1(EPOLL_CLOEXEC);
```

Comment:
`epoll_create(0)` is more portable

Reply:
@yosefe  the ```epoll_create``` 's manul page says: ```Since Linux 2.6.8, the size argument is ignored, but must be greater than zero;```
I'll use ```epoll_create(1)``` here

Reply:
right, so epoll_create(1)